preprocessing/filtered_innerouter_snsc_mousekidney.ipynb

```diff
@@ -210,90 +210,6 @@
     "bulk_adata = bulk_adata.transpose()"
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    pca_095 = PCA(n_components=0.95)
</pre><p>hmmm how is the number of components less than 1? what does this mean?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/greenelab/rnaseq_translator/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='136'/>